src/adapters/MeetingsJSONAdapter.test.js

```diff
@@ -43,6 +43,17 @@ describe('Meetings JSON Adapter', () => {
     );
   });
 
+  test('getMeeting() renders the local media if localVideo property is defined', (done) => {
+    window.navigator.mediaDevices = {
+      getUserMedia: () => jest.fn(Promise.resolve()),
+    };
+    meetingID = 'localVideo';
+    meetingsJSONAdapter.getMeeting(meetingID).subscribe((data) => {
+      expect(data).toEqual(meetings[meetingID]);
```

Comment:
If we resolve a mock string as mentioned above, we could test it here:

```suggestion
      expect(data.localVideo).toEqual('mock-stream');
```

Reply:
yeah. that's cool